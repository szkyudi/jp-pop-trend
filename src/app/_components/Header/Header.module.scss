@import '../../../lib/styles/variables.scss';

.header {
  background: $color-primary;
}

.container {
  height: 48px;
  display: flex;
  align-items: center;
  justify-content: center;
}

.title {
  font-size: 20px;
  font-weight: bold;
  color: #fff;
}
