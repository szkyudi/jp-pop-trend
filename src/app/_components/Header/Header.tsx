import s from './Header.module.scss'
import { Container } from '@/lib/components/Contianer'

export function Header() {
  return (
    <header className={s.header}>
      <Container>
        <div className={s.container}>
          <h1 className={s.title}>JP-POP-TREND</h1>
        </div>
      </Container>
    </header>
  )
}
