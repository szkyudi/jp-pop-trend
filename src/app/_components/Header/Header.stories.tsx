import { Header } from './Header'
import { Meta, StoryObj } from '@storybook/react'

const meta: Meta<typeof Header> = {
  component: Header,
}

export const Default: StoryObj<typeof meta> = {}

export default meta
