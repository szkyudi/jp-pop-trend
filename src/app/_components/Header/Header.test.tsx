import * as stories from './Header.stories'
import { render } from '@testing-library/react'
import { composeStories } from '@storybook/react'

const { Default } = composeStories(stories)

it('表示に変更がないこと', () => {
  const { container } = render(<Default />)

  expect(container).toMatchSnapshot()
})
