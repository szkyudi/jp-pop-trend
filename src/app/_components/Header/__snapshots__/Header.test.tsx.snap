// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`表示に変更がないこと 1`] = `
<div>
  <header
    class="_header_146992"
  >
    <div
      class="_container_450716"
    >
      <div
        class="_container_146992"
      >
        <h1
          class="_title_146992"
        >
          JP-POP-TREND
        </h1>
      </div>
    </div>
  </header>
</div>
`;
